systems.csv

```diff
@@ -236,6 +238,7 @@ US,Sobi Long Beach,"Long Beach, NY",sobi_long_beach,http://sobilongbeach.com/,ht
 US,Spartanburg BCycle,"Spartanburg, SC",bcycle_spartanburg,https://spartanburg.bcycle.com,https://gbfs.bcycle.com/bcycle_spartanburg/gbfs.json
 US,Spin Chicago,"Chicago, IL",spin chicago,https://www.spin.pm/,https://web.spin.pm/api/gbfs/v1/chicago/gbfs.json
 US,Spin Detroit,"Detroit, MI",spin detroit,https://www.spin.pm/,https://web.spin.pm/api/gbfs/v1/detroit/gbfs
+US,Spin Louisville,"Louisville, KY",spin louisville,https://web.spin.pm/api/gbfs/v1/louisville/free_bike_status
```

Comment:
This should point to the auto-discovery endpoint https://web.spin.pm/api/gbfs/v1/louisville/gbfs

Reply:
Agreed and fixed, sorry I missed it.